tests/perf-test/OptionsSelector.perf-test.tsx

```diff
@@ -35,8 +38,11 @@ jest.mock('../../src/components/withNavigationFocus', () => (Component) => {
     return WithNavigationFocus;
 });
 
-const generateSections = (sectionConfigs) =>
-    _.map(sectionConfigs, ({numItems, indexOffset, shouldShow = true}) => ({
+type GenerateSectionsProps = Array<{numItems: number; indexOffset: number; shouldShow?: boolean}>;
+
+const generateSections = (sectionConfigs: GenerateSectionsProps) =>
+    sectionConfigs.map(({numItems, indexOffset, shouldShow = true}) => ({
```

Comment:
"Configs" doesn't add any information to this variable name, and we have a type, so we don't need any qualifiers like configs, or data, or info, or anything like that
```suggestion
const generateSections = (section: GenerateSectionsProps) =>
    section.map(({numItems, indexOffset, shouldShow = true}) => ({
```